src/main/jbake/content/blog/2017/07/opentracing-jaxrs.adoc

```diff
@@ -0,0 +1,136 @@
+= OpenTracing JAX-RS Instrumentation
+Pavol Loffay
+2017-7-10
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, tracing, jaeger, opentracing, jax-rs
+
+In the previous http://www.hawkular.org/blog/2017/06/9/opentracing-spring-boot.html[demo]
+we have demonstrated how to instrument Spring Boot app
+using http://opentracing.io[OpenTracing] a vendor-neutral standard for distributed tracing.
+In this article we are going to instrument JAX-RS and show how to trace business layer and add
```

Comment:
In this article we are going to instrument a Java API for RESTful Web Services (JAX-RS), and show you how to trace the business layer...."